ml-agents/mlagents/envs/brain.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, visual_observation, vector_observation, text_observations, me
         self.previous_vector_actions = vector_action
         self.previous_text_actions = text_action
         self.action_masks = action_mask
+        self.custom_output =  custom_output
```

Comment:
I think this should be `self.custom_outputs`

Reply:
I definitely see the logic of that, but I wonder if it will be confusing since `custom_outputs` is a scalar and rather than a repeated field.

Reply:
I do not understand this. Shouldn't `custom_output` be a list of custom_output ? If there are `N` agents in the environment each sending a custom_output at the same step, the BrainInfo should contain all of them.

Reply:
Ah you're right; I misinterpreted your comment since it is scalar protobuf message attached to agent but is indeed list of those message on the `BrainInfo` instance and so the name change makes sense.